libs/designsystem/src/lib/components/form-field/input/input.component.ts

```diff
@@ -31,6 +36,13 @@ export class InputComponent implements OnChanges {
     }
   }
 
+  @Input() size: InputSize = InputSize.large;
+
+  @HostBinding('class')
+  get _cssClass() {
+    return [this.size];
+  }
```

Comment:
If size is mapped 1:1 with css class consider applying `@HostBinding` directly on `size` property. Having a separate property for this is meaningful if merging 2 or more props to css class or when mapping property value to different css classes...
```suggestion
  @HostBinding('class')
  @Input() size: InputSize = InputSize.large;
```

Reply:
Cleaner, like it 👍